lib/init-transforms/entry.js

```diff
@@ -0,0 +1,6 @@
+const run = (config, answer) => {
+	config.entry('index').add(answer).end();
```

Comment:
Unless you are going to be doing more chaining here, you can leave off `.end()`, since you are just returning `config` anyway.